identity-service/src/awsSNS.js

```diff
@@ -92,37 +92,29 @@ const publishPromisified = _promisifySNS('publish')
 const deleteEndpoint = _promisifySNS('deleteEndpoint')
 
 async function publish (message, userId, tx, playSound = true) {
-  bufferMessages(message, userId, tx, PUSH_NOTIFICATIONS_BUFFER, playSound)
+  await addNotificationToBuffer(message, userId, tx, PUSH_NOTIFICATIONS_BUFFER, playSound)
```

Comment:
@hareeshnagaraj we shouldn't deploy your other PR until these awaits go in there somehow either through a patch or this PR

Reply:
thats fine, the changes are already on stage but not prod